Add more detail to CodeArtifact instructions (#22)
Brightspace/lms-version-actions: match-lms-release/README.md

## NPM Package Deployment

> If you'd like the action to deploy your package to NPM or CodeArtifact, set the `NPM` option to `true`.

Reviewer: Perhaps this is more technically correct?
```suggestion
If you'd like the action to publish your package with NPM (including to CodeArtifact), set the `NPM` option to `true`.
```
